### PR TITLE
Update majority of images in "Your first 2d game"

### PR DESCRIPTION
**Pull request by Piralein**
Update majority of images to new layout and change some of the text referring to images.

Not included:
- new-project-button.png
- folder-content.png
- player_coll_shape.png
- mob_animations.gif
- draw_path2d.gif
- ui_anchor.png
- start_button_shortcut.png
- dodge_preview.gif

I'm not sure if using the FontVariation is really needed here, but I tried to stay as close to the original as possible.

(superseeds #6635, sorry :/ )

### CODE REVIEW

File: getting_started/first_2d_game/03.coding_the_player.rst [hunk 1]
```diff
@@ -144,13 +144,17 @@ Click on *Project -> Project Settings* to open the project settings window and
...
+We need to assign a key to this action. Click the "+" icon on the right, to
+open the event manager window.
 
-.. image:: img/input-mapping-add-key.png
+.. image:: img/input-mapping-add-key.webp
+
+Use the "Listening for Input..." field to filter for the "Keyboard keys - right" event and
+add it with the "ok" button.
+
+.. image:: img/input-mapping-event-configuration.webp
```
Comment (skyace65) [inline getting_started/first_2d_game/03.coding_the_player.rst:157]:
```suggestion
The "Listening for Input..." field should automatically be selected. Press the "right" key on your keyboard, and the menu should look like this now.

.. image:: img/input-mapping-event-configuration.webp

Select the "ok" button. The "right" key is now associated with the ``move_right`` action.
```

**Review by skyace65 — CHANGES_REQUESTED** [review]
Needs a little work but overall this is a fantastic PR! The image quality is perfect and exactly in line with the new guidelines.

File: getting_started/first_2d_game/02.player_scene.rst [hunk 2]
```diff
@@ -18,7 +18,7 @@ rule, a scene's root node should reflect the object's desired functionality -
 what the object *is*. Click the "Other Node" button and add an :ref:`Area2D
 <class_Area2D>` node to the scene.
 
-.. image:: img/add_node.png
+.. image:: img/add_node.webp
```
[the remark this answers is not in the document]
Comment (Piralein) [reply getting_started/first_2d_game/02.player_scene.rst:21]: Yes, I forgot to turn on 3d, fixed.^^"

**Comment by Piralein** [comment]
I renamed the font pictures to something more descriptive.
I also removed the `Player` part of the signal connection, it is not part of the autogenerated name.
(please check the cpp part carefully, I'm still not really familiar with it)

**Review by skyace65 — APPROVED** [review]
All good now! Thanks a ton for this! Just for future reference we like to keep everything in a PR as one commit. I'm going to squash and merge this now but keep it in mind for future reference. If you don't know how to do it we have a guide on [this page](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html).

**Comment by Piralein** [comment]
I wanted to keep it separate for easier review and squash after approval.
Didin't expect it to be this fast. 😄

**Pull request merged**